MULTIPLATFORM.md

```diff
@@ -0,0 +1,325 @@
+`ios/build.gradle` and add an `expectedBy` dependency in the same manner as in the Android project:
+
+    apply plugin: 'konan'
+
+    // Specify targets to build the framework: iOS and iOS simulator
+    konan.targets = ['iphone', 'iphone_sim']
+
+    konanArtifacts {
+        // Declare building into a framework.
+        framework('iosGreeting') {
+            // The multiplatform support is disabled by default.
+            enableMultiplatform true
+        }
+    }
+
+    dependencies {
+        // Specify dependency on a common project for Kotlin multiplatform build
+        expectedBy project(':greeting:common')
+    }
+
+### 3. Android application
+
+Now we can create an Android application which will use the library we implemented on the previous step. Open Android
+Studio and create a new project in the `androidApp` directory. Android Studio generates all necessary files and
+directories so we only need to add a dependency on our library. There are 2 actions we need to do:
+
+1. Add dependency on the library. To do this just open `androidApp/app/build.gradle` and add the following snippet in
+the `dependencies` script block:
+
+    ```
+    implementation 'org.greeting:android:1.0'
+    ```
+2. Include `greeting` build in the Android Studio project as a part of
+[composite build](https://docs.gradle.org/current/userguide/composite_builds.html). To do this, add the
+following line in `androidApp/settings.gradle`:
+
+    ```
+    includeBuild '../greeting'
+    ```
+    Now dependencies of the application can be resolved in artefacts built by `greeting`. You also may publish the
+    Android part of `greeting` into some Maven repo and get it from there. In this case you don't need to set up
+    the composite build.
+
+After these steps we can access our library as any other Kotlin code:
+
+    import org.greeting.*
+
+    /* ... */
+
+    fun foo() {
+        println(Greeting().greeting())
+    }
+
+**TODO: Describe including `greeting` as a  subproject of the AS project too.**
+
+### 4. iOS application
+
+As said above the multiplatform library can also be used in iOS applications. The general approach here is the same as
+in case of an Android application: we create a separate XCode project and add the library as a Framework. But we need
+to make some additional steps here.
+
+Unlike Android Studio XCode doesn't use Gradle, so we cannot just add the library as a dependency. Instead we need to
+create a new Framework in the XCode project and then replace its default build phases with a custom one that delegates
+building the framework to Gradle.
+
+To do this, make the following steps:
+
+1. Create a new XCode project using `iosApp` as a root directory for it.
+2. Add a new framework in the project. Go `File` -> `New` -> `Target` -> `Cocoa Touch Framework`. Specify the same
+framework name as in `greeting/ios/build.gradle`: `iosGreeting`).
```

Comment:
We have to rename that iosGreeting to just `Greeting` as well (if we want to do it). This is just a a reminder to don't forget it..

Reply:
Done